### PR TITLE
Fixed broken CI/CD

### PR DESCRIPTION
**Pull request by marcomicera**
### Related issues

- #5
- #6

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -176,5 +176,5 @@ jobs:
           kustomize edit set image endocodeci/qmstr-master:$SHA_TAG
           kustomize edit set image endocodeci/qmstr-client-mvn:$SHA_TAG
           kustomize edit set image endocodeci/pom-patch:$SHA_TAG
-          kubectl apply -k .
+          kustomize build . | kubectl apply -f -
```
Comment (marcomicera) [inline .github/workflows/main.yml:179]: Cool, as I thought. Thanks @somayeh-najafi!

**Comment by marcomicera** [comment]
Looks good to me. @somayeh-najafi anything else that could be improved?

There are two warning that _might_ cause problems in the future:

```
Thank you for using setup-gcloud Action. GoogleCloudPlatform/github-actions/setup-gcloud has been deprecated, please switch to google-github-actions/setup-gcloud.

@ deploy-to-dev: .github#L1


Ubuntu-latest workflows will use Ubuntu-20.04 soon. For more details, see https://github.com/actions/virtual-environments/issues/1816

@ Build, push, and deploy: .github#L1
```

Should we try to switch to `google-github-actions/setup-gcloud` so that we can avoid problems in the future?

**Comment by somayeh-najafi** [comment]
@marcomicera  Oh yeah, Sure I forgot to change it. I'll do.

**Comment by marcomicera** [comment]
> Should we use a specific version of Ubuntu or latest is fine?

I don't know what we can specify there, so I cannot answer.

**Pull request merged**